templates/show-worklist.html

```diff
@@ -15,6 +16,7 @@
 
     $(document).ready(function(){
 
+    	// making the nav active
```

Comment:
Nit: no need for this comment? It adds nothing to what I can read in the actual code right below.